prime_directive/core/scribe.py

```
@@ -41,30 +41,37 @@ async def generate_sitrep(
-        openai_timeout_seconds (float): Timeout in seconds for the OpenAI
-            request.
-        openai_max_tokens (int): Maximum number of tokens for the OpenAI
-            request.
-        api_url (str): The Ollama API endpoint.
-        timeout_seconds (float): Timeout in seconds for the Ollama request.
-        max_retries (int): Maximum number of retries for the Ollama request.
-        backoff_seconds (float): Backoff time in seconds between retries.
-
+    Generate a concise SITREP (situation report) summarizing repository state, recent terminal output, and optional human/task context.
+    
+    Constructs a prompt from repo_id, git_state, terminal_logs, active_task, and human_* fields, then requests a short (2–3 sentence) SITREP with an immediate next step from the configured provider and model. By default uses Ollama; can use OpenAI as the primary provider or as a fallback. When OpenAI is used and db_path is provided, the function will check monthly budget and log estimated token usage and cost.
```

Ruff (0.14.8)

Docstring contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF002)